examples/07_advanced/plot_beta_series.py

```diff
@@ -102,6 +103,8 @@
 # :func:`~nilearn.glm.first_level.first_level_from_bids`.
 import matplotlib.pyplot as plt
 
+print("Fit model")
```

Comment:
adding some print statement because the example takes a long time to run and this helps know what is happening